packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -574,8 +574,21 @@ export class Cluster extends Resource implements ICluster {
  * ECS-optimized Windows version list
  */
 export enum WindowsOptimizedVersion {
-  SERVER_2019 = '2019',
-  SERVER_2016 = '2016',
+  SERVER_20H2_CORE = 'Windows_Server-20H2-English-Core',
+  SERVER_2004_CORE = 'Windows_Server-2004-English-Core',
+  SERVER_2019_FULL = 'Windows_Server-2019-English-Full',
```

Comment:
We can't use enums where there are multiple different enum entries with the same string values (this one and `SERVER_2016_FULL`). This unfortunately causes unexpected behavior when using the CDK in non-typescript languages. 

However, this kind of fixes itself if you revert the `SERVER_2019` and `SERVER_2016` entries back to their old values to not break existing users.

Reply:
Hi @madeline-k, this isn't my PR, however the string values for `SERVER_2016_FULL` and `SERVER_2019_FULL` have different strings, one for 2016 and one for 2019, but I could be misunderstanding the issue here.

Reply:
Sorry, my comment wasn't clear. the string for SERVER_2019_FULL is the same as the string for SERVER_2019. And SERVER_2016_FULL is the same as SERVER_2016.